vignettes/IntroToAnnotationPackages.Rmd

```diff
@@ -0,0 +1,383 @@
+---
+title: "AnnotationDbi: Introduction To Bioconductor Annotation Packages"
+author: 
+  - name: "Marc Carlson"
+  - name: "Beryl Kanali"
+    affiliation: "Converted vignette from Sweave to Rmarkdown"
+date: "Last modified: November 2022; Compiled: `r format(Sys.time(), '%B %d , %Y')`"
+
+vignette: >
+  %\VignetteIndexEntry{1. Introduction To Bioconductor Annotation Packages}
+  %\VignetteDepends{org.Hs.eg.db, TxDb.Hsapiens.UCSC.hg19.knownGene, EnsDb.Hsapiens.v75}
+  %\VignetteKeywords{annotation, database}
+  %\VignettePackage{AnnotationDbi}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document
+---
+
+```{r  label = fig:dbtypes, echo = FALSE, fig.align="center", fig.cap = "Annotation Packages: the big picture", out.width = '60%'}
+knitr::include_graphics("databaseTypes.png")
+```
+
+```{r echo=FALSE, include=FALSE}
+if (!require("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+
+BiocManager::install("BiocStyle", force = TRUE)
+```
+
+
+```{r include=FALSE}
+library(knitr)
+opts_chunk$set(tidy=FALSE)
+```
+
+
+```{r include=FALSE}
+library(BiocStyle)
+```
+
+*Bioconductor* provides extensive annotation resources. These can be
+*genecentric*, or *genome centric*. Annotations can be provided in packages
+curated by *Bioconductor*, or obtained from web-based resources. This vignette
+is primarily concerned with describing the annotation resources that are
+availableas packages. More advanced users who wish to learn about how to make
+newannotation packages should see the vignette titled "Creating select
+Interfacesfor custom Annotation resources" from the 
```

Comment:
There should be space between `Interfaces` and `for` Please correct all the instances such as these and let me know I will continue my review after you correct these instances.